controllers/topics.controller.js

```diff
@@ -7,7 +11,21 @@ exports.getTopics = (req, res) => {
 };
 
 exports.getArticles = (req, res) => {
```

Comment:
I see you have topics.controller.js but include article controllers in here - consider separating these out into different files at this stage i.e. make an article.controller.js :)

Reply:
Can do, chief!